pipelines/coaddColumnValidate.yaml

```diff
@@ -0,0 +1,17 @@
+description: |
+  Tier 1 DRP metrics to assess table column validity
+tasks:
+  validateObjectTableCore:
+    class: lsst.analysis.tools.tasks.ObjectTableTractAnalysisTask
+    config:
+      bands: ['g', 'r', 'i', 'z', 'y']
+      connections.outputName: objectTableCore
```

Comment:
This should probably be `connections.outputName: objectTableColumnValidate` right since coaddQualityCore.yaml uses objectTableCore as an output name